recipe/build.sh

```diff
@@ -22,6 +22,7 @@ autoreconf --install
     --enable-hepmc2=$PREFIX \
     --enable-lhapdf=$PREFIX \
     --with-sqlite3=$PREFIX \
+    --without-python \
```

Comment:
This isn't working given

```
configure: WARNING: unrecognized options: --without-python
```

Reply:
sherpa is using `autconf`? I thought it had cmake support via something like `-DSHERPA_ENABLE_PYTHON=false` or something

Reply:
@kratsg CMake support was added in [Sherpa `v3.0.0`](https://gitlab.com/sherpa-team/sherpa/-/blob/58abdd66176f2750e5c1bfd83402bb6ebc40b3b8/CMakeLists.txt). I intentionally started with the last `v2.x` release so that if people were stuck on `v2` for whatever reason they would still have a release on conda-forge to use. `v2` is `autotools` based.